lucille-core/src/main/java/com/kmwllc/lucille/stage/ParseFloats.java

```diff
@@ -23,8 +26,14 @@ public ParseFloats(Config config) {
   public Iterator<Document> processDocument(Document doc) throws StageException {
     if (doc.has(field)) {
       String value = doc.getString(this.field);
-      List<Float> floats = Arrays.stream(value.replaceAll("\\[", "").replaceAll("\\]", "").replaceAll(",", "").split(" "))
-          .map(v -> Float.parseFloat(v)).collect(Collectors.toList());
+      List<Float> floats;
+      try {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
both the ObjectMapper and the TypeReference should be private static final vars on the class. We don't need to create them with each call to processDocument()

Reply:
makes sense